src-ui/src/locale/messages.de_DE.xlf

```diff
@@ -209,7 +209,7 @@
           <context context-type="sourcefile">src/app/components/manage/tag-list/tag-list.component.html</context>
           <context context-type="linenumber">1</context>
         </context-group>
-        <target state="needs-translation">Tags</target>
+        <target state="translated">Schlagwörter</target>
```

Comment:
Don't think we should be translating "Tags"...

Reply:
Me no speak German so totally defer to you / others. If thats the case it needs to be done in Crowdin.

Reply:
This was kindly translated by @DavidMoraisFerreira on Crowdin.

I'm not sure if this is the right place to review translations. Wouldn't that be easier on Crowdin?

@DavidMoraisFerreira what is your opinion on this?

Reply:
Yes you can chat on Crowdin but I suppose either is fine if all parties are here. Also if theres a concensus I can make a change in Crowdin (this time or in general)

Reply:
@bauerj "Tags" is very prominent in German indeed. I retracted my translation of this Term from Crowdin.
As to where the review should take place, Crowdin does have useful features for reviewing translations. But in any case, the PR is where it really matters :)